### PR TITLE
migrate from plugin builder v3 to v4

### PR DESCRIPTION
**Pull request by ahcai**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,46 +1,17 @@
 {
   "name": "plugin-agent-autoresponse",
-  "version": "0.0.0",
+  "version": "0.0.2",
```
Comment (ahcai) [inline package.json:3]: hmm, not sure why it changed the version

Comment (ktalebian) [reply package.json:3]: You probably did a deploy?

Comment (ahcai) [reply package.json:3]: oh maybe i did when we were testing a while back. i'll change it back

Comment (Amstrong) [reply package.json:3]: Yes, it's time to do it. It does not allow Deploy and when updating the version it fails.

Comment (ahcai) [reply package.json:3]: this will be merged tomorrow!

**Review by ktalebian — APPROVED** [review]

**Pull request merged**